generate.js

```diff
@@ -449,6 +449,10 @@ let update_output = function () {
   // }
 
   scenario.innerText = output;
+
+  let file_contents = `Feature: Description of broad purpose for all scenarios in here${ output }`;
```

Comment:
Hmm... not sure why this is happening, but in the downloaded file for me, the "Feature" text is prepended by a space. Not essential to fix or anything, but it's just weird.

Reply:
Made #27, though I checked just now and this it doesn't break tests. Still, a bit weird. Thanks for catching it.